test_scripts/Smoke/commonSmoke.lua

```diff
@@ -266,16 +88,17 @@ function commonSmoke.updatePreloadedPT()
       hmi_levels = { "BACKGROUND", "FULL", "LIMITED" }
     }
   end
-  pt.policy_table.app_policies["0000001"] = commonSmoke.cloneTable(pt.policy_table.app_policies.default)
+  pt.policy_table.app_policies["0000001"] = utils.cloneTable(pt.policy_table.app_policies.default)
   pt.policy_table.app_policies["0000001"].groups = { "Base-4", "NewTestCaseGroup" }
   pt.policy_table.app_policies["0000001"].keep_context = true
   pt.policy_table.app_policies["0000001"].steal_focus = true
-  utils.tableToJsonFile(pt, preloadedFile)
+  SDL.PreloadedPT.set(pt)
 end
 
-function commonSmoke.preparePreloadedPTForRC()
-  local preloadedFile = commonPreconditions:GetPathToSDL() .. preloadedPT
-  local pt = utils.jsonFileToTable(preloadedFile)
+function common.preparePreloadedPTForRC()
+  isPreloadedUpdated = true
+  SDL.PreloadedPT.backup()
```

Comment:
@jacobkeeler This could happen in case if this function is called several times in one script. Fortunately there are no such scripts in Smoke.
Anyway for robustness such update worth to be made.